src/pl/ogladajanime/src/eu/kanade/tachiyomi/animeextension/pl/ogladajanime/extractors/CdaPlExtractor.kt

```diff
@@ -0,0 +1,129 @@
+package eu.kanade.tachiyomi.animeextension.pl.ogladajanime.extractors
+
+import eu.kanade.tachiyomi.animesource.model.Video
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.POST
+import eu.kanade.tachiyomi.util.asJsoup
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import okhttp3.Headers
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.RequestBody.Companion.toRequestBody
+import uy.kohesive.injekt.injectLazy
+import java.net.URLDecoder
+import java.nio.charset.StandardCharsets
+
+class CdaPlExtractor(private val client: OkHttpClient) {
```

Comment:
Instead of copy+pasting the code from another extension, it would be better to move the extractor to a shared lib instead.